pdr_backend/lake/test/test_fetch_ohlcv.py

```diff
@@ -43,6 +46,53 @@
     ]
 }
 
+mock_bad_token_dydx_response_1 = {
+    "errors": [
```

Comment:
Move these dydx test variables to be located right before `test_safe_fetch_ohlcv_dydx()`.